app/src/main/java/org/oppia/android/app/mydownloads/MyDownloadsActivity.kt

```diff
@@ -17,20 +19,30 @@ class MyDownloadsActivity : InjectableAppCompatActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     activityComponent.inject(this)
-    myDownloadsActivityPresenter.handleOnCreate()
```

Comment:
Please update the KDoc for this class at top.

Reply:
it's there already `/** The activity for displaying [MyDownloadsFragment]. */`

Reply:
Yes but we don't have a `MyDownloadsFragment ` file.

Reply:
Ah, missed it

Reply:
done